hub/core/chunk_engine/write.py

```diff
@@ -14,17 +14,32 @@
 from .read import read_index_map, read_tensor_meta, tensor_exists
 
 
+def tuple_to_list(shape: Tuple):
+    shapeArray = np.asarray(shape)
```

Comment:
What's wrong with `list(some_tuple)`?

Reply:
`list(some_tuple)` was giving an error earlier. It works perfectly now. I will make this change.